app/src/main/res/values/strings.xml

```diff
@@ -462,21 +462,21 @@
 
     <string name="settings_notification_heading">Choose notification action</string>
     <string name="settings_notification_desc">Action buttons to show with the RethinkDNS notification: %1$s</string>
-    <string name="settings_notification_desc1">Pause / Stop (default)</string>
-    <string name="settings_notification_desc2">DNS / DNS + Firewall</string>
+    <string name="settings_notification_desc1">Pause / stop (default)</string>
+    <string name="settings_notification_desc2">DNS / DNS + firewall</string>
     <string name="settings_notification_desc3">None</string>
 
     <string name="settings_notification_dialog_title">Choose notification action</string>
-    <string name="settings_notification_dialog_option_1">\nPause / Stop (default):\nShows two buttons, to quickly pause and stop RethinkDNS.</string>
-    <string name="settings_notification_dialog_option_2">\nDNS / DNS + Firewall:\nShows two buttons to quickly switch between the battery-saver DNS mode and the regular DNS + Firewall mode.</string>
-    <string name="settings_notification_dialog_option_3">\nNone:\nShows no button.\n</string>
+    <string name="settings_notification_dialog_option_1">\nPause / stop (default):\nTwo buttons, to quickly pause and stop RethinkDNS.</string>
+    <string name="settings_notification_dialog_option_2">\nDNS / DNS + Firewall:\nTwo buttons to quickly switch between the battery-saver DNS mode and the regular DNS + firewall mode.</string>
+    <string name="settings_notification_dialog_option_3">\nNone:\nNo buttons.\n</string>
 
     <string name="settings_selected_theme">Current theme: %1$s</string>
 
-    <string name="settings_protocol_translation_dns_inactive">Switch to DNS/DNS+Firewall mode to enable this setting</string>
+    <string name="settings_protocol_translation_dns_inactive">Switch to DNS / DNS + firewall mode to turn on this setting</string>
```

Comment:
`enable` / `disable` provide better visual cue is why we use it... though, if `turn on` / `turn off` translate well, then that's fine too.

Reply:
That is the idea. Also reduced complexity.